specification/vmware/Microsoft.AVS.Management/client.tsp

```diff
@@ -0,0 +1,215 @@
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-core/invalid-final-state" "MUST CHANGE ON NEXT UPDATE"
+  createDnsService is Microsoft.AVS.WorkloadNetworkDnsServices.create;
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  updateDnsService is Microsoft.AVS.WorkloadNetworkDnsServices.update;
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-operation"
+  deleteDnsService is Microsoft.AVS.WorkloadNetworkDnsServices.delete;
+
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  listDnsZones is Microsoft.AVS.WorkloadNetworkDnsZones.list;
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  getDnsZone is Microsoft.AVS.WorkloadNetworkDnsZones.get;
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-core/invalid-final-state" "MUST CHANGE ON NEXT UPDATE"
+  createDnsZone is Microsoft.AVS.WorkloadNetworkDnsZones.create;
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  updateDnsZone is Microsoft.AVS.WorkloadNetworkDnsZones.update;
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-operation"
+  deleteDnsZone is Microsoft.AVS.WorkloadNetworkDnsZones.delete;
+
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "Can not change existing operationId."
+  listPublicIPs is Microsoft.AVS.WorkloadNetworkPublicIps.list;
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "Can not change existing operationId."
+  getPublicIP is Microsoft.AVS.WorkloadNetworkPublicIps.get;
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-core/invalid-final-state" "MUST CHANGE ON NEXT UPDATE"
+  createPublicIP is Microsoft.AVS.WorkloadNetworkPublicIps.create;
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "Can not change existing operationId."
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-operation"
+  deletePublicIP is Microsoft.AVS.WorkloadNetworkPublicIps.delete;
+}
```

Comment:
Re-configure the operation groups.

The writing of the ops is not that, but all these `#suppress` is nightmare.

Reply:
I cannot use this `client` and `operationGroup` in client.tsp.

There is `@singleton WorkloadNetwork` resource. The correct handling of the route path depends on the `@armProviderNamespace` decorator which cannot apply to client.tsp
https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-resource-manager/src/namespace.ts#L240-L246